x-pack/solutions/search/plugins/search_homepage/public/components/ai_search_capabilities/semantic_search.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+
+import { AISearchWorkflow } from './ai_search_workflow';
+import { useAssetBasePath } from '../../hooks/use_asset_base_path';
+
+export const SemanticSearch: React.FC = () => {
+  const assetBasePath = useAssetBasePath();
+
+  const semanticSearchText = {
+    image: `${assetBasePath}/semantic_search.svg`,
+    imageAlt: i18n.translate('xpack.searchHomepage.aiSearchCapabilities.semanticSearch.imageAlt', {
+      defaultMessage: 'Semantic Search',
+    }),
+    heading: i18n.translate('xpack.searchHomepage.aiSearchCapabilities.semanticSearch.title', {
+      defaultMessage: 'Enhance search accuracy with advanced semantic capabilities.',
+    }),
+    subheading: i18n.translate(
+      'xpack.searchHomepage.aiSearchCapabilities.semanticSearch.description',
+      {
+        defaultMessage:
+          "Leverage the semantic_text field along with Elastic's advanced ELSER machine learning model for enhanced data analysis.",
+      }
+    ),
+    featureBullets: [
+      i18n.translate('xpack.searchHomepage.aiSearchCapabilities.semanticSearch.firstLine', {
+        defaultMessage: 'Use Elastic’s inference service or connect your own model provider',
+      }),
+      i18n.translate('xpack.searchHomepage.aiSearchCapabilities.semanticSearch.SecondLine', {
+        defaultMessage: 'Default chunking strategies or customize your own',
```

Comment:
>        defaultMessage: 'Default chunking strategies or customize your own',

The "customize your own" option didn't seem to be available in the "semantic search" index creation workflow, so is this misleading to mention here?